pkg/cmd/experimental/sync-groups/interfaces.go

```diff
@@ -0,0 +1,33 @@
+package group
+
+import (
+	"github.com/go-ldap/ldap"
+
+	userapi "github.com/openshift/origin/pkg/user/api"
+)
+
+// LDAPGroupLister lists the groups that need to be synced by a job. The groups are represented with
```

Comment:
specify ldap versus openshift groups.  I don't care how much longer it makes the comments, this is inscrutable.